README.md

```diff
@@ -22,9 +22,7 @@ A simple 1:1 WebRTC video application.
 
 # Pre-Requisites
 
-In order to use the Bandwidth API, users need to set up the appropriate application in their [Bandwidth Dashboard](https://dashboard.bandwidth.com/) and create [API credentials](https://dev.bandwidth.com/guides/accountCredentials.html#top). WebRTC must be enabled for your account to run this sample. Please check with your account manager to ensure you are provisioned for WebRTC.
```

Comment:
A Voice or Messaging application isn't required for this sample, which is why I removed mention of it